src/common/Tile/styled.js

```diff
@@ -45,7 +45,7 @@ export const Poster = styled.img`
   border-radius: 5px;
   background: ${({ theme }) => theme.colors.Silver};
   object-position: center center;
-  object-fit: scale-down;
+  object-fit: cover;
```

Comment:
To samo musimy niżej powtórzyć dla widoku mobilnego, bo tam teraz ucina nam obrazek ;)

Reply:
Oki :) pewnie przeoczyłem

Reply:
sprawdzam i nie widzę żeby coś ucinało 🤔

Reply:
ale między cover a scale-down też nie widzę różnicy na mobilnym widoku więc w sumie nie ma potrzeby dodawać jakiegokolwiek object-fit do media queries tylko niech będzie zawsze cover